skglm/datafits/multi_task.py

```diff
@@ -89,3 +89,52 @@ def full_grad_sparse(self, X_data, X_indptr, X_indices, Y, XW):
                     XjTXW[t] += X_data[i] * XW[X_indices[i], t]
             grad[j, :] = (XjTXW - self.XtY[j, :]) / n_samples
         return grad
+
+
+spec_QuadraticGroup = [
```

Comment:
Group datafits are not multitask (the response `y` is still a vector, not a matrix) so the code rather belongs to a newly created `datafits/group.py`